client/src/main/java/io/pravega/client/tables/KeyValueTableConfiguration.java

```diff
@@ -50,10 +50,22 @@
      * The number of bytes for the Secondary Key. This value cannot be changed after the Key-Value Table has been created.
      *
      * @param secondaryKeyLength The number of bytes for the Secondary Key.
-     * @return The number of bytes for the Primary Key.
+     * @return The number of bytes for the Secondary Key.
      */
     private final int secondaryKeyLength;
 
+    /**
+     * The rollover size for table segment in LTS.
+     *
+     * The default value for this field is 0.
+     * If default value is passed down to the server, a non-zero value defined in the server
+     * will be used for the actual rollover size.
+     *
+     * @param rolloverSizeBytes The rollover size for the table segment in LTS.
+     * @return The rollover size for the table segment in LTS.
+     */
+    private final long rolloverSizeBytes;
```

Comment:
Could this value be chosen by the Controller? 
How often would the user need to set a different rollover size for segments of different Key Value Tables?